CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+We welcome community support with both pull requests and reporting bugs. Please
+don't hesitate to jump in.
+
+## Review others' work
+
+Check out the list of outstanding pull requests if there is something you might
+be interested in. Maybe somebody is trying to fix that stupid bug that bothers
+you. Review the PR. Do you have any better ideas how to fix this problem? Let us
+know.
+
+## Issues
+
+The issue tracker is the preferred channel for bug reports, features requests
+and submitting pull requests.
+
+_Note: Occasionally issues are opened that are unclear, or we cannot verify them. When the issue author has not responded to our questions for verification within 7 days then we will close the issue._
+
+## Tests
+
+All commits that fix bugs or add features need appropriate unit tests.
+
+## Code Style
+
+Please adhere to the current code styling. We have included an `.editorconfig`
+at the repo's root to facilitate uniformity regardless of your editor. See the
+[editor config site][editorconfig] for integration details.
+
+We use [ESLint][eslint] for all JavaScript Linting. There should be no linting
+errors and no new warnings for new work. You are welcome to configure your
+editor to use ESLint or the `npm test` command will run unit tests and the
+linter.
+
+## Visual Changes
+
+When making a visual change, if at all feasible please provide screenshots
+and/or screencasts of the proposed change. This will help us to understand the
+desired change easier.
+
+## Docs
+
+Please update the docs with any API changes, the code and docs should always be
+in sync.
+
+## Breaking changes
+
+We follow semantic versioning rather strictly.
+
+## Collaborators
```

Comment:
I think this section is incredibly useful if we plan to bring on other
contributers.

On Fri, May 6, 2016 at 3:38 PM Anson Wayman <notifications@github.com>
wrote:

> In CONTRIBUTING.md
> <https://github.com/jira-node/node-jira-client/pull/62#discussion_r62394582>
> :
>
> > +## Visual Changes
> > +
> > +When making a visual change, if at all feasible please provide screenshots
> > +and/or screencasts of the proposed change. This will help us to understand the
> > +desired change easier.
> > +
> > +## Docs
> > +
> > +Please update the docs with any API changes, the code and docs should always be
> > +in sync.
> > +
> > +## Breaking changes
> > +
> > +We follow semantic versioning rather strictly.
> > +
> > +## Collaborators
>
> Whoops. Didn't realize I had nuked this. I didn't plan on having one
> considering the size of this project.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/jira-node/node-jira-client/pull/62/files/44307d8ce80f468e1bcf16ef6fbee70ce287ba07#r62394582>
>